packages/driver/src/cypress/error_messages.ts

```diff
@@ -298,7 +298,7 @@ export default {
       docsUrl: 'https://on.cypress.io/contains',
     },
     length_option: {
-      message: `${cmd('contains')} cannot be passed a \`length\` option because it will only ever return 1 element.`,
```

Comment:
This error message was incorrect - you can in fact pass a length option. It just has to be 1. :woman_shrugging: Not a change in `.contains()`s behavior - it's always been this way.